misk/src/main/kotlin/misk/tracing/backends/zipkin/ZipkinBackendModule.kt

```diff
@@ -0,0 +1,40 @@
+package misk.tracing.backends.zipkin
+
+import brave.Tracing
+import brave.opentracing.BraveTracer
+import com.google.common.util.concurrent.Service
+import com.google.inject.Provides
+import io.opentracing.Tracer
+import misk.config.AppName
+import misk.inject.KAbstractModule
+import misk.inject.addMultibinderBinding
+import misk.inject.to
+import misk.tracing.TracingService
+import zipkin2.reporter.AsyncReporter
+import zipkin2.reporter.okhttp3.OkHttpSender
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+class ZipkinBackendModule(val config: ZipkinBackendConfig) : KAbstractModule() {
+  override fun configure() {
+    binder().addMultibinderBinding<Service>().to<TracingService>()
```

Comment:
why does each impl need to do this? shouldn't the `TracerModule` do this for you?

Reply:
Good catch. After the refactor to always install a tracer even there isn't one in the config (e.g. `NoopTracer`) this can be moved into the Tracing module. Will fix.